CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 # Change Log
 All notable changes to COMMIT will be documented in this file.
 
+
+## [1.5.0] - 2020-10-23
+
+### Added
+- core.load_confidence_map(): Load the confidence map from nii file
```

Comment:
Description does not match the actual code

Reply:
Changed